siriuspy/siriuspy/opticscorr/chrom.py

```diff
@@ -389,7 +389,7 @@ def _callback_estimate_chrom(self, pvname, value, **kws):
         for fam in self._psfams:
             fam_idx = self._psfams.index(fam)
```

Comment:
maybe this can be replaced by ``for fam_idx, fam in enumerate(self._psfams):``, what do you think?

Reply:
it seems to be clearer, I'll replace that!